MvvmCross/Platforms/Ios/Presenters/MvxIosViewPresenter.cs

```diff
@@ -474,10 +467,19 @@ private Task<bool> ShowPopoverViewControllerChild(UIViewController viewControlle
             presentationController.Delegate = new MvxPopoverPresentationControllerDelegate(this);
 
             PopoverViewController = viewController;
-            await viewHost.PresentViewControllerAsync(viewController, attribute.Animated).ConfigureAwait(true);
+
+            var parentViewController = GetParentViewController();
+            await parentViewController.PresentViewControllerAsync(viewController, attribute.Animated).ConfigureAwait(true);
             return true;
         }
 
+        private UIViewController GetParentViewController()
```

Comment:
I know it is late, but can you make methods like these protected so that subclasses can utilize them? This class is notoriously difficult to subclass because of things like that

Reply:
The PR is already closed. Everyone can contribute by creating a new PR. You're welcome to create one.

PS. Personally, I don't think there's a profit by changing the behavior of GetParentViewController(). The less you change, the less you break :)